Indicators/VolumeWeightedMovingAverage.cs

```diff
@@ -0,0 +1,87 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator computes the Volume Weighted Moving Average (VWMA)
+    /// </summary>
+    public class VolumeWeightedMovingAverage : TradeBarIndicator, IIndicatorWarmUpPeriodProvider
+    {
+
+        private IndicatorBase<IndicatorDataPoint> _rollingSumS { get; }
+        private IndicatorBase<IndicatorDataPoint> _rollingSumV { get; }
```

Comment:
Smees this PR is missing to address previous reviews, 
- XML description improvement
- privates should be fields, no need to be properties
- `..SumS/..SumV` should be expanded to their complete names
- please push `IsReady` above constructor
- variable `_sumV` in method scope name is off codying style
- missing brackets
-